src/applications/caregivers/helpers.js

```diff
@@ -19,7 +20,97 @@ export const medicalCenterLabels = Object.keys(caregiverFacilities).reduce(
 );
 
 // Turns the facility list for each state into an array of strings
-export const medicalCentersByState = _.mapValues(
+const medicalCentersByState = _.mapValues(
   val => val.map(center => center.code),
   caregiverFacilities,
 );
+
+// transforms forData to match fullSchema structure for backend submission
+const submitTransform = (formConfig, form) => {
+  // checks for optional chapters using ssnOrTin
+  const hasSecondaryOne =
+    form.data.secondaryOneSsnOrTin === undefined ? null : 'secondaryOne';
+
+  const hasSecondaryTwo =
+    form.data.secondaryTwoSsnOrTin === undefined ? null : 'secondaryTwo';
+
+  // creates chapter objects by matching chapter prefixes
+  const buildChapterSortedObject = (data, dataPrefix) => {
+    // check to make sure there is a keyName
+    if (dataPrefix === null) return {};
+    const keys = Object.keys(data);
+
+    // matches prefix to fullSchema chapter object labels/keys
+    const getChapterName = key => {
+      switch (key) {
+        case 'veteran':
+          return 'veteran';
+        case 'primary':
+          return 'primaryCaregiver';
+        case 'secondaryOne':
+          return 'secondaryCaregiverOne';
+        case 'secondaryTwo':
+          return 'secondaryCaregiverTwo';
+        default:
+          return null;
+      }
+    };
```

Comment:
You can make this an object
```
const chapterNames = Object.freeze({
  'veteran': 'veteran',
  'primary': 'primaryCaregiver',
  'secondaryOne': 'secondaryCaregiverOne',
  'secondaryTwo': 'secondaryCaregiverTwo',
}
```
then when you need it just use:
```
chapterNames[key]
```

Reply:
I am going to have to look into this at another time, I am having issues null checking it, one of the nice parts about the switch is that default statement which helps reject undefined. Will look into this feedback and see if I can get it working with less code down the road.

Reply:
Oh yeah, LOL I just recommended this same thing LOL